src/ngx_http_lua_kong_error_log.c

```diff
@@ -0,0 +1,183 @@
+/**
+ * Copyright 2019-2022 Kong Inc.
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *    http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "ngx_http_lua_kong_common.h"
+
+
+static ngx_array_t *error_log_append_var_indexes;
+
+typedef struct {
+    ngx_str_t   name;
+    ngx_uint_t  index;
+} var_elt_t;
+
+
+static u_char *
+ngx_http_log_error_handler(ngx_http_request_t *r, ngx_http_request_t *sr,
```

Comment:
Should have  the prefix `ngx_http_lua_kong_`?

Reply:
We should add a comment like `copy from ngx_http_request.c:ngx_http_log_error_handler()`.